_po/ja/how-to/laravel.po

```diff
@@ -507,6 +747,12 @@ msgid ""
 "            <div class=\"bg-white overflow-hidden shadow-sm sm:rounded-lg\">\n"
 "                <div class=\"p-2 bg-white border-b border-gray-200\">"
 msgstr ""
+"```php\n"
+"<x-guest-layout>\n"
+"    <div class=\"py-2\">\n"
+"        <div class=\"max-w-7xl mx-auto sm:px-6 lg:px-8\">\n"
+"            <div class=\"bg-white overflow-hidden shadow-sm sm:rounded-lg\">\n"
+"                <div class=\"p-2 bg-white border-b border-gray-200\">"
```

Comment:
Probably, this is needless.
Because this test is same as before translation text.